scripts/jupyter/heir_play/heir_opt.py

```diff
@@ -28,6 +28,7 @@ def run_binary(self, line, cell):
             [os.path.abspath(self.binary_path)] + shlex.split(line),
             input=cell,
             text=True,
+            capture_output=True,
```

Comment:
Is it really the best to use `capture_output=True` here? Capturing the output here means that the entire stdout and stderr are read into memory,  which might lead to the binary to produce a large amount of output.  Without this we could allow the output to be streamed directly to the console.

Reply:
What console? This is intended for use in a notebook environment, and this change is specifically to work with Google colab, which would be unusable without this change, until they update their dependencies.